exporter/lokiexporter/example/otel-collector-config.yml

```diff
@@ -16,7 +16,6 @@ processors:
   # information, see
   # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md
   memory_limiter:
-    ballast_size_mib: 683
```

Comment:
This [deprecated parameter](https://github.com/open-telemetry/opentelemetry-collector/pull/4696/files) was also preventing the otel container to execute, please let me know if there is a better option than removing it.

<img width="1413" alt="Screen Shot 2022-04-04 at 11 00 17 PM" src="https://user-images.githubusercontent.com/611147/161632684-1d148cba-874d-48ed-bed7-09eb7609be8d.png">

Reply:
The recommendation is to use the [ballast extension](https://github.com/open-telemetry/opentelemetry-collector/tree/main/extension/ballastextension)